src/native/public/mono/metadata/details/class-types.h

```diff
@@ -17,6 +17,11 @@ typedef struct _MonoClassField MonoClassField;
 typedef struct _MonoProperty MonoProperty;
 typedef struct _MonoEvent MonoEvent;
 
+typedef struct MonoVarianceSearchEntry {
+	MonoClass *klass;
+	int interface_offset;
+} MonoVarianceSearchEntry;
```

Comment:
This seems to be unused.

Also even if it was used it doesn't need to be in the public API headers.

Reply:
I deleted it in my local branch, I'm not sure why the push failed. Will push again